<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListManagerDialog</class>
 <widget class="QDialog" name="ListManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>313</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_Lists">
        <property name="text">
         <string>Liste</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbb_Lists"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="lst_Entries"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_MoveUp">
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_MoveDown">
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Add">
        <property name="text">
         <string>Hinzufügen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Remove">
        <property name="text">
         <string>Entfernen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="bb_ListManagerDialog">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_ListManagerDialog</sender>
   <signal>accepted()</signal>
   <receiver>ListManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_ListManagerDialog</sender>
   <signal>rejected()</signal>
   <receiver>ListManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
